docs/user/subprojects.rst

```diff
@@ -1,30 +1,30 @@
-Subprojects
-===========
+Organizing multiple projects in one site: Subprojects
+=====================================================
 
 Projects can be configured in a nested manner, by configuring a project as a
 *subproject* of another project. This allows for documentation projects to share
 a search index and a namespace or custom domain, but still be maintained
 independently.
 
-For example, a parent project, ``Foo`` is set up with a subproject, ``Bar``. The
-documentation for ``Foo`` will be available at:
+This is useful for:
+
+* Organizations that need all their projects visible in one documentation portal or landing page
+* Projects that document and release several packages or extensions
+* Organizations or projects that want to have a common search function for several sets of documentation
+
+For example, a parent project, ``foo`` is set up with a subproject, ``bar``. The
+documentation for ``foo`` will be available at:
```

Comment:
I'd love to see a more real example here. Perhaps `example-project` and `example-api` or something?

Reply:
I used `example-project` and `plugin` :)